### PR TITLE
Update PPP for Jingjing

### PR DESCRIPTION
**Pull request by zhaojj2209**
Partially addresses #271.
Drafted Project Portfolio Page for Jingjing.

### CODE REVIEW

**Review by yongping827 — DISMISSED** [review]
LGTM

File: docs/team/zhaojj2209.md [hunk 1]
```diff
@@ -5,27 +5,41 @@ title: Zhao Jingjing's Project Portfolio Page
...
+  * What it does: With the addition of the monthly budget, the finance tracker allows the user to set their desired monthly expense limit and monthly savings goal, and calculates the user's remaining budget and current savings.
+  * Justification: This feature significantly enhances the capabilities of the finance tracker by allowing the user to track their progress in their savings, and paves the way for the application's analytics feature.
+  * Highlights: This feature required heavy use of the observer design pattern as all values in the monthly budget have to be observed by the UI so that the values and calculations can be updated immediately upon any changes.
 
-* **New Feature**:
-  * What it does:
-  * Justification:
-  * Highlights:
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **Major Enhancement**: Revamped the finance tracker's `find` command.
```
Comment (wltan) [inline docs/team/zhaojj2209.md:19]: Should `Enhancement` be lowercase to remain consistent with the rest of the bullet points?

File: docs/team/zhaojj2209.md [hunk 2]
```diff
@@ -24,22 +24,22 @@ Fine$$e is a desktop finance tracker that allows tertiary students to better man
...
   * User Guide:
-    * Updated user guide to reflect commands for v1.2 (Pull request #114)
-    * Updated user guide for `find`, `set-expense-limit` and `set-savings-goal` commands for v1.3 (Pull request #176)
+    * Updated user guide to reflect commands for v1.2. (Pull request #114)
+    * Updated user guide for `find`, `set-expense-limit` and `set-savings-goal` commands for v1.3. (Pull request #176)
   * Developer Guide:
-    * Updated user stories and use cases in developer guide (Pull request #13)
-    * Updated find transactions and monthly budget features in developer guide (Pull request #120, #144)
+    * Updated user stories and use cases in the developer guide. (Pull request #13)
+    * Updated find transactions and monthly budget features in the developer guide. (Pull request #120, #144)
```
Comment (ianyong) [inline docs/team/zhaojj2209.md:40]: Should we use the plural form when there are multiple pull requests?

**Review by ianyong — DISMISSED** [review]
LGTM!

**Review by wltan — APPROVED** [review]
LGTM!

**Review by ianyong — APPROVED** [review]
LGTM!

**Pull request merged**